dss-api.yml

```diff
@@ -687,10 +687,10 @@ definitions:
         description: Replica this query is monitoring. One of aws, gcp, or azure.
         type: string
         enum: [aws, gcp, azure]
-      creator_uid:
-        type: integer
-        format: int
-        description: User ID who created this subscription.
+      owner:
```

Comment:
Since nobody but the owner can see notifications for now, I question the need for including this field.